### PR TITLE
generate_packageTest: Refactor sys.argv mock

### PR DESCRIPTION
**Pull request by chriscoyfish**
Fixes: https://github.com/coala-analyzer/coala-bears/issues/597

### CODE REVIEW

File: tests/generate_packageTest.py [hunk 1]
```diff
@@ -90,25 +91,41 @@ class mainTest(unittest.TestCase):
...
+        main()
+        with open(self.CSS_BEAR_SETUP_PATH) as fl:
+            setup_py = fl.read()
+        self.assertNotIn(VERSION, setup_py)
+        self.assertIn(VERSION[:VERSION.find("dev")], setup_py)
+
     @patch('bears.generate_package.perform_upload')
     def test_upload(self, call_mock):
-        sys.argv = ["generate_package.py", "--upload"]
+        sys.argv.append("--upload")
```
Comment (Adrianzatreanu) [inline tests/generate_packageTest.py]: hey this commit looks amazing, but i think the changes in lines 121, 128 and 94 are unrelated to this commit and should be changed to another commit accordingly :)

Comment (chriscoyfish) [inline tests/generate_packageTest.py]: I'm happy to omit moving the argv to setUp, but my new tests also need that set, so it seemed cleaner to move it to setUp and save some LOCs. Please let me know if this seems reasonable?

Comment (Adrianzatreanu) [inline tests/generate_packageTest.py]: it sounds really good, but to fix this, you could first move the set in a commit, and THEN do your changes. this way your changes will use the set and you will also have correct commits. is this okay :) ?
